linkerd/core/src/main/scala/io/buoyant/linkerd/telemeter/UsageDataTelemeter.scala

```diff
@@ -117,13 +125,16 @@ private[telemeter] object UsageDataTelemeter {
     counters.toSeq
   }
 
+  type HttpRouter = { def identifier: Option[Seq[{ def kind: String }]] }
```

Comment:
what is costly?  the reflexive calls?  these are actually used "in the serving path" but that path is only exercised once per hour (or when requested from the admin endpoint)